pkg/webhook/pod/util.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	FAIL_TO_FIND_TIDB_COMPONENT_OWNER_STATEFULSET = "failed to find owner statefulset for pod[%s/%s]"
+	FailToFindTidbComponentOwnerStatefulset = "failed to find owner statefulset for pod[%s/%s]"
```

Comment:
The error message has nothing to do with `TidbComponent`.
BTW, is it necessary to export here?

Reply:
Yes, in case of deleting pod, we need to know whether the owner statefulset of tidb component ( not all the pods ) is whether existed. If not existed, the request to delete pod shall be admitted.